src/components/dialog-full-screen/dialog-full-screen-test.stories.tsx

```diff
@@ -141,6 +143,39 @@ Nested.parameters = {
   },
 };
 
+export const WithFullWidthHeader: StoryType = {
```

Comment:
**praise**: nice one for creating the story in the latest CSF 3.0 format. Ideally, this should be the format we use going forward 👍🏼